packages/sampleOS/build.yaml

```diff
@@ -1,16 +1,19 @@
-requires:
+join:
```

Comment:
this uses final images instead of the build containers, we could have also written this as: 

```yaml
requires:
- category: "system"
  name: "cos"
  version: ">=0"

copy:
- package: 
      category: "app"
      name: "sampleOSService"
      version: ">=0"
  source: /usr/bin/sample...
  destination: ..
```

Reply:
https://github.com/rancher-sandbox/cos-toolkit-sample-repo/commit/1e9e33a7feb600ecef5948d137659f95e8e31b0b

Reply:
Mentioned it in a comment :+1: